src/github-web-worker.js

```diff
@@ -167,6 +167,11 @@ const api = {
         content: JSON.stringify(value)
       };
 
+      if (currentFiles && currentFiles.hasOwnProperty(this.oldBackupFileName)) {
+        // found old backup file, it will be deleted
+        files[this.oldBackupFileName] = null;

```

Comment:
I will accept the PR but it ain't deleting the old file, so it created the `project-viewer-default.json` and kept the `project-viewer.json`.